docs/reference/aggregations/bucket/histogram-aggregation.asciidoc

```diff
@@ -11,19 +11,10 @@ To make this more formal, here is the rounding function that is used:
 
 [source,java]
 --------------------------------------------------
-rem = value % interval
-if (rem < 0) {
-    rem += interval
-}
-bucket_key = value - rem
+bucket_key = Math.floor((value - offset) / interval) * interval + offset
 --------------------------------------------------
 
-From the rounding function above it can be seen that the intervals themselves **must** be integers.
-
-WARNING: Currently, values are cast to integers before being bucketed, which
-might cause negative floating-point values to fall into the wrong bucket. For
-instance, `-4.5` with an interval of `2` would be cast to `-4`, and so would
-end up in the `-4 <= val < -2` bucket instead of the `-6 <= val < -4` bucket.
+The `interval` must be a positive decimal, while the `offset` must be a decimal in `[0 - interval[`.
```

Comment:
I've never seen the `[0, interval[` notation before! I had no idea it was in an [ISO standard](https://en.wikipedia.org/wiki/ISO_31-11)! I used a wikipedia link because the standard is paywalled I think... That standard was superceded by [ISO 80000-2](http://www.ise.ncsu.edu/jwilson/files/mathsigns.pdf). That link is to a scan.... Anyway, it says to use the `[0, interval)` notation but that the `[0, interval[` notation "is also used".

Now that I've done the 20 minutes of research I think you can use whatever you want. I learned the `[0, interval)` notation in school and thought that the backwards `[` on the end was a typo. Some folks will think that too but they too will learn something new that day!